TriblerGUI/widgets/settingspage.py

```diff
@@ -60,6 +61,17 @@ def initialize_settings_page(self):
         self.window().partially_empty_tokens_button.clicked.connect(self.partially_empty_tokens)
         self.window().log_location_chooser_button.clicked.connect(self.on_choose_log_dir_clicked)
 
+        checkbox_style = get_checkbox_style()
+        for checkbox in [self.window().family_filter_checkbox, self.window().channel_autocommit_checkbox,
+                         self.window().always_ask_location_checkbox, self.window().developer_mode_enabled_checkbox,
+                         self.window().use_monochrome_icon_checkbox, self.window().download_settings_anon_checkbox,
+                         self.window().download_settings_anon_seeding_checkbox, self.window().lt_utp_checkbox,
+                         self.window().watchfolder_enabled_checkbox, self.window().allow_exit_node_checkbox,
+                         self.window().credit_mining_enabled_checkbox, self.window().developer_mode_enabled_checkbox,
+                         self.window().checkbox_enable_network_statistics, self.window().checkbox_enable_resource_log,
+                         self.window().download_settings_add_to_channel_checkbox]:
+            checkbox.setStyleSheet(checkbox_style)
```

Comment:
We should instead use CSS stylesheets in the `.ui` file. It is especially important as we are going to make the GUI's CSS  totally configurable. Is it possible to move this thing to the stylesheet?

Reply:
The way our ui is structured, it does not seem to support class-based styles so I needed either to repeat the styles for each checkboxes or apply programmatically. We should refactor it when write global CSS for Tribler.